### PR TITLE
cleanup warnings

### PR DESCRIPTION
**Pull request by gandrewstone**

### CODE REVIEW

File: src/script/sign.cpp [hunk 1]
```diff
@@ -104,6 +104,8 @@ static bool SignStep(const BaseSignatureCreator& creator, const CScript& scriptP
     case TX_MULTISIG:
         scriptSigRet << OP_0; // workaround CHECKMULTISIG bug
         return (SignN(vSolutions, creator, scriptPubKey, scriptSigRet));
+    case TX_LABELPUBLIC:  // These are OP_RETURN unspendable outputs so they should never be an input that needs signing
+        return false;
```
Comment (ptschip) [inline src/script/sign.cpp]: I would just move this up to line 80...then there's no need for another "return false"

Comment (sickpig) [reply src/script/sign.cpp]: agreed

File: src/test/excessiveblock_test.cpp [hunk 2]
```diff
@@ -21,8 +21,8 @@ BOOST_AUTO_TEST_CASE(rpc_excessive)
     BOOST_CHECK_NO_THROW(CallRPC("getminingmaxblock"));
 
     BOOST_CHECK_THROW(CallRPC("setexcessiveblock not_uint"), runtime_error);
-    BOOST_CHECK_THROW(CallRPC("setexcessiveblock 1000000 not_uint"), boost::bad_lexical_cast);
-    BOOST_CHECK_THROW(CallRPC("setexcessiveblock 1000000 -1"), boost::bad_lexical_cast);
+    BOOST_CHECK_THROW(CallRPC("setexcessiveblock 4000000 not_uint"), boost::bad_lexical_cast);
```
Comment (sickpig) [inline src/test/excessiveblock_test.cpp:24]: why 4000000?

Comment (gandrewstone) [reply src/test/excessiveblock_test.cpp:24]: needs to be above 2 mb and this gives us a little room... its arbitrary

File: src/script/sign.cpp (hunk 1 above)
Comment (gandrewstone) [reply src/script/sign.cpp]: ok

**Pull request merged**